com.unity.shadergraph/Editor/Data/Nodes/UV/ParallaxOcclusionMappingNode.cs

```diff
@@ -88,15 +88,19 @@ public void GenerateNodeFunction(FunctionRegistry registry, GenerationMode gener
             var heightmap = GetSlotValue(kHeightmapSlotId, generationMode);
 
             // We first generate components that can be used by multiple POM node
-            registry.ProvideFunction("Unique", s =>
+            registry.ProvideFunction("PerPixelHeightDisplacementParam", s =>
             {
                 s.AppendLine("struct PerPixelHeightDisplacementParam");
                 using (s.BlockSemicolonScope())
                 {
-                    s.AppendLine("$precision2 uv;");
+                    s.AppendLine("float2 uv;");
```

Comment:
can't use $precision if the "function"/struct is not actually generic.
And we can't make the struct generic because it's used by code in an include file.
Given this is parallax occlusion mapping node, we're probably ok to assume uv coords are full 32 bit.